.github/workflows/build.yml

```diff
@@ -25,6 +25,8 @@ on:
   schedule:
     # run this workflow every day at 7am UTC
     - cron:  '0 7 * * *'
+    # run this workflow every midnight Tuesday
```

Comment:
Minor comment - I would suggest making this run Monday morning UTC so there's a little more time to investigate any issues before our TSC meeting in case there's anything to discuss. Also would prefer to avoid any times that aren't immediately obvious (ie does 'midnight Tuesday' mean Monday/Tuesday or Tuesday/Wednesday?).

Reply:
Done!